Organic/AmpAffiliateInjector.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Organic;
+
+use AMP_Base_Sanitizer;
+use DOMDocument;
+use DOMXPath;
+
+class AmpAffiliateInjector extends \AMP_Base_Sanitizer {
+    private $organic;
+
+    public function sanitize() {
+        try {
+            $this->organic = Organic::getInstance();
+            $this->site_organic_domain = $this->organic->getSiteOrganicDomain();
```

Comment:
The organic object doesn't have `getSiteOrganicDomain` method. I removed it since the SDK has inlined configuration.

Reply:
I think this means I'll need to do create a new platform endpoint to execute a GraphQL query for the site domain. Does that sound right? I want to make sure you don't have an easier approach.

Reply:
We have an enpoint that is used to inline configs, you can use that one:
https://github.com/orgnc/platform/blob/9d1c8b0143b77cf2bf90cff6363bb62f24cbe380/backend/sdk/api/rest.py#L13

Reply:
Ah, awesome. Thank you!